src/templates/organization.js

```diff
@@ -21,6 +21,12 @@ function getLogoImage({ logo, photos, categories }) {
     categories.find(c => c.cover) || categories.find(c => c?.parent?.cover)
   return logo || photos[0] || cat?.cover || cat?.parent.cover
 }
+function getEditUrl({ data, org }) {
+  const { capitalEditFormUrl, organizationEditFormUrl } = data.site.siteMetadata
+  const url =
+    data.type === "organization" ? organizationEditFormUrl : capitalEditFormUrl
```

Comment:
What is `data.type` here? I think we should be checking if `role` includes `"Capital"`

Reply:
good call